src/frontend/src/pages/SettingsPage/UserSettings/UserSettingsEdit.tsx

```diff
@@ -73,6 +76,57 @@ const UserSettingsEdit: React.FC<UserSettingsEditProps> = ({ currentSettings, on
           )}
         />
       </Grid>
+
+      <Grid item>
```

Comment:
can you make it so that on a large screen each of these items is on the same line, maybe make address a little bit bigger than the other ones